app/views/hyrax/my/_collection_action_menu.html.erb

```diff
@@ -1,39 +1,38 @@
-<%# OVERRIDE Hyrax v3.4.2 Adjust collection deletion permissions %>
+<%# OVERRIDE Hyrax v5.0.0rc2 Adjust collection deletion permissions %>
 <% id = collection_presenter.id %>
 <% ul_id = 'collection-action-dropdown-ul-' + id %>
 
 <div class="btn-group">
   <button class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown" type="button" id="dropdownMenu_<%= id %>" aria-haspopup="true" aria-expanded="false" aria-controls="<%= ul_id %>">
     <span class="sr-only"><%= t("hyrax.dashboard.my.sr.press_to") %> </span>
-    <%= t("hyrax.dashboard.my.action.select") %> <span class="caret" aria-hidden="true"></span>
+    <%= t("hyrax.dashboard.my.action.select") %>
```

Comment:
Curious about removing this, but non-blocker as it's likely a button beautifier.